sql/planbuilder/dml.go

```diff
@@ -111,6 +111,7 @@ func (b *Builder) buildInsert(inScope *scope, i *ast.Insert) (outScope *scope) {
 	dest := destScope.node
 
 	ins := plan.NewInsertInto(db, plan.NewInsertDestination(sch, dest), srcScope.node, isReplace, columns, onDupExprs, ignore)
+	ins.SkipSourceAnalyze = !requiresAnalyze
```

Comment:
Should this come after validate below?

Reply:
not sure it makes a difference? value is only used at analyze time